services/api/handlers/projects.js

```diff
@@ -254,6 +254,27 @@ exports.patch = {
   }
 };
 
+exports.delShallow = function(request, reply) {
+  request.server.methods.projects.remove(
+    [
+      request.params.project
+    ],
+    function(err, result) {
+      if(err) {
```

Comment:
the double wrapping error sounds like it is being stubbed somewhere and restore isn't song called on it. might be an existing bug